next/src/validation/schema.ts

```diff
@@ -224,6 +224,8 @@ export function validateSchema(
       errors.push({
         path: [...path, key],
         validation: 'required',
+        schema: schema?.properties?.[key] || schema,
```

Comment:
I don't think `schema?.properties?.[key]` is correct. It should always be `schema` because the `required` is a constraint on the current `schema` object schema and not it's properties.

Reply:
that is, the logic for the checkbox error message is expecting the schema to match the actual checkbox schema…

Reply:
Ah, I see. `getErrorMessage` needs the schema of the property that's required to see if `schema['x-jsf-presentation']?.inputType === 'checkbox'`. Feels a little bit wrong looking at just the validation error itself...
But let's keep it as it is for now. I can't think of a better way to do it right now.